pkg/queue/bounded_queue_test.go

```diff
@@ -156,3 +157,173 @@ func (s *consumerState) assertConsumed(expected map[string]bool) {
 	}
 	assert.Equal(s.t, expected, s.snapshot())
 }
+
+func TestResizeUp(t *testing.T) {
+	q := NewBoundedQueue(2, func(item interface{}) {
+		fmt.Printf("dropped: %v\n", item)
+	})
+
+	var firstConsumer, secondConsumer, releaseConsumers sync.WaitGroup
+	firstConsumer.Add(1)
+	secondConsumer.Add(1)
+	releaseConsumers.Add(1)
+
+	released, resized := false, false
+	q.StartConsumers(1, func(item interface{}) {
+		if !resized { // we'll have a second consumer once the queue is resized
```

Comment:
I didn't find the second consumer

Reply:
Around line 200:

https://github.com/jaegertracing/jaeger/blob/a7819b895930fc85727750f2a00cf5b21bc8cbb7/pkg/queue/bounded_queue_test.go#L197-L200